search_queries_range.go

```diff
@@ -9,79 +9,84 @@ package elastic
+	gt        interface{}
+	gte       interface{}
+	lt        interface{}
+	lte       interface{}
+	timeZone  string
+	boost     *float64
+	queryName string
+	format    string
 }
 
 // NewRangeQuery creates and initializes a new RangeQuery.
 func NewRangeQuery(name string) *RangeQuery {
-	return &RangeQuery{name: name, includeLower: true, includeUpper: true}
+	return &RangeQuery{name: name}
 }
 
-// From indicates the from part of the RangeQuery.
+// From indicates the `gte` part of the RangeQuery.
 // Use nil to indicate an unbounded from part.
 func (q *RangeQuery) From(from interface{}) *RangeQuery {
-	q.from = from
-	return q
+	return q.Gte(from)
 }
 
-// Gt indicates a greater-than value for the from part.
-// Use nil to indicate an unbounded from part.
-func (q *RangeQuery) Gt(from interface{}) *RangeQuery {
-	q.from = from
-	q.includeLower = false
-	return q
+// To indicates the `lte` part of the RangeQuery.
+func (q *RangeQuery) To(to interface{}) *RangeQuery {
+	return q.Lte(to)
 }
 
-// Gte indicates a greater-than-or-equal value for the from part.
-// Use nil to indicate an unbounded from part.
-func (q *RangeQuery) Gte(from interface{}) *RangeQuery {
-	q.from = from
-	q.includeLower = true
+// Gt indicates a greater-than value.
+// Use nil to indicate an unbounded gt part.
+func (q *RangeQuery) Gt(gt interface{}) *RangeQuery {
+	q.gt = gt
 	return q
 }
 
-// To indicates the to part of the RangeQuery.
-// Use nil to indicate an unbounded to part.
-func (q *RangeQuery) To(to interface{}) *RangeQuery {
-	q.to = to
+// Gte indicates a greater-than-equal value.
+// Use nil to indicate an unbounded gte part.
+func (q *RangeQuery) Gte(gte interface{}) *RangeQuery {
+	q.gte = gte
 	return q
 }
 
-// Lt indicates a less-than value for the to part.
-// Use nil to indicate an unbounded to part.
-func (q *RangeQuery) Lt(to interface{}) *RangeQuery {
-	q.to = to
-	q.includeUpper = false
+// Lt indicates a less-than value.
+// Use nil to indicate an unbounded `lt` part.
+func (q *RangeQuery) Lt(lt interface{}) *RangeQuery {
+	q.lt = lt
 	return q
 }
 
-// Lte indicates a less-than-or-equal value for the to part.
-// Use nil to indicate an unbounded to part.
-func (q *RangeQuery) Lte(to interface{}) *RangeQuery {
-	q.to = to
-	q.includeUpper = true
+// Lte indicates a less-than-or-equal value.
+// Use nil to indicate an unbounded `lte` part.
+func (q *RangeQuery) Lte(lte interface{}) *RangeQuery {
+	q.lte = lte
 	return q
 }
 
 // IncludeLower indicates whether the lower bound should be included or not.
 // Defaults to true.
 func (q *RangeQuery) IncludeLower(includeLower bool) *RangeQuery {
-	q.includeLower = includeLower
+	if includeLower {
+		q.lte = q.lt
+		q.lt = nil
+	} else {
+		q.lt = q.lte
+		q.lte = nil
+	}
 	return q
 }
 
 // IncludeUpper indicates whether the upper bound should be included or not.
 // Defaults to true.
 func (q *RangeQuery) IncludeUpper(includeUpper bool) *RangeQuery {
-	q.includeUpper = includeUpper
+	if includeUpper {
```

Comment:
it's unclear if this method (and [IncludeLower](https://github.com/olivere/elastic/pull/214/files#diff-e0951bfd14ee4008141d6c8a228a4178R70) should now do nothing or if it should mutate the query.